app-mock/src/androidTest/java/pl/dybczak/githubusers/test/UsersPageTest.kt

```diff
@@ -0,0 +1,49 @@
+package pl.dybczak.githubusers.test
+
+import androidx.test.rule.ActivityTestRule
+import androidx.test.runner.AndroidJUnit4
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import pl.dybczak.githubusers.robots.UsersPageRobot
+import pl.sulej.users.view.UsersActivity
+
+@RunWith(AndroidJUnit4::class)
+class UsersPageTest {
+
+    private val searchPageRobot = UsersPageRobot()
+
+    @get:Rule
+    val activityRule = ActivityTestRule(UsersActivity::class.java)
+
+    @Test
+    fun assertAppNameDisplayed() {
```

Comment:
We can use that kind of naming (convention 4. in this article [it's about unit tests, but applies here as well] https://dzone.com/articles/7-popular-unit-test-naming) for test cases - it describes given feature/design property, and assert* names are better suitable for assertion methods of robots.
```suggestion
    fun displayAppName() {
```